app/bundles/FormBundle/Helper/FormFieldHelper.php

```diff
@@ -220,7 +220,10 @@ public function populateField($field, $value, $formName, &$formHtml)
                 }
                 break;
             case 'checkboxgrp':
-                if (!is_array($value)) {
+                if (is_string($value) && strrpos($value, '|') > 0) {
+                    $value = explode('|', $value);
+                }
+                else if (!is_array($value)) {
                     $value = [$value];
```

Comment:
The code looks good! Just to keep the code style, could you please write the `else if` at the same line as `}`? Like this:
```
} else if (!is_array($value)) {
```
The best for the future PRs would be if you'd make the pre-commit script working so it fix CS issues for you:
https://github.com/mautic/mautic/blob/staging/.github/CONTRIBUTING.md#coding-standards